src/pages/Dashboard/inputComponent.definitions.ts

```diff
@@ -0,0 +1,10 @@
+export interface ReusableInputProps {
+  name: string;
+  register: any;
+  errors: any;
```

Comment:
Since we're going to use a special resolver for errors, you can set errors as a `string` (and you can change it to `error`). Don't worry about the way hook form typically does errors